### PR TITLE
update redirects

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by karlkfi** [comment]
The FROM (left side) needs to be a page, but the TO (right side) should be the canonical url (directory preferred).

**Comment by joel-hamill** [comment]
@karlkfi fixed

**Comment by karlkfi** [comment]
Please remove the `index.html` from the right sides. The canonical url is the directory. The web server automatically serves index.html when accessing the directory.

**Comment by joel-hamill** [comment]
@karlkfi isn't that what the source has now? https://cl.ly/103S1L3I461H

**Comment by karlkfi** [comment]
yeah. it didn't show pushed till 8am tho.

File: redirects [hunk 1]
```diff
@@ -26,4 +26,4 @@
 /overview/components/index.html /docs/latest/overview/components/
 /support /community/
 /usage/cli/install/index.html /docs/latest/usage/cli/install/
-/docs/1.7/usage/tutorials/ruby-on-rails/index.html /docs/1.7/usage/tutorials/
+/docs/1.7/usage/tutorials/ruby-on-rails/index.html /docs/latest/usage/tutorials/
```
Comment (karlkfi) [inline redirects]: Did you intentionally redirect from 1.7 docs to latest? This might be undesirable behavior once 1.8 docs are available.

Comment (joel-hamill) [inline redirects]: nope, let me fix that

Comment (joel-hamill) [inline redirects]: actually, in most cases, yes. the original link isn't versioned, so why version the redirect? will this redirect file be versioned eventually?

**Comment by karlkfi** [comment]
looks good. merging

**Comment by karlkfi** [comment]
please double check that the redirs work as expected in the staging env, once CI pushes it.

File: redirects (hunk 1 above)
Comment (karlkfi) [inline redirects]: I'm not sure what you're asking. I'm also not sure why this redirect exists. What links to it?

**Comment by joel-hamill** [comment]
@karlkfi these do not work. i'll have to look into tomorrow. please do not push to live.

**Comment by karlkfi** [comment]
What doesn't work?

**Comment by joel-hamill** [comment]
@karlkfi i was trying these locally. thomas informed this wouldn't work. also, in talking to thomas he recommended adding the full version path. that being said, these may actually work.

**Pull request merged**